app/assets/stylesheets/components/_button.scss

```diff
@@ -39,9 +39,19 @@
   background: linear-gradient(#147EFF, #0F67DA);
   color: white;
 }
-.btn-create a {
+
+.button {
   background: $purple;
-  padding: 12px 24px;
+  padding: 10px 18px;
   color: $white;
-  border-radius: 10px;
+  border-radius: 5px;
+  text-decoration: none;
+  opacity: 0.9;
```

Comment:
Not sure if you need to put all of the above attributes in a nested .button class. Why not put them all in the .btn-create a selector?

Reply:
Actually I changed btn-create class name as button then implement all the button with same class name. Instead of btn-create as a class name I think I might be better if I use more general name of it. But we can write all of them btn-create as well.

Reply:
Ah ok, now i understand. When the line is red in in the GitHub pull request it means the line has been deleted then? Ok then I'm all on board with your changes.